zuul.d/projects.yaml

```diff
@@ -137,7 +137,7 @@
 
 - project:
     name: kubernetes-sigs/kind
-    periodic:
+    periodic-6/18:
```

Comment:
Why kind jobs have to be triggered 4 times in one day? for most of projects, 2 times is enough